include/envoy/http/codec.h

```diff
@@ -36,6 +36,11 @@ const char MaxResponseHeadersCountOverrideKey[] =
 
 class Stream;
 
+enum class ErrorCode {
```

Comment:
👍 will update. Initially this enumerated all the error codes [here](https://http2.github.io/http2-spec/#ErrorCodes), which are shared with RST_STREAM. But we've since undone that and there's already a separate Envoy-internal enum for stream resets.